examples/adjoints/adjoints_ex6/element_postprocess.C

```diff
@@ -0,0 +1,62 @@
+// General libMesh includes
+#include "libmesh/libmesh_common.h"
+#include "libmesh/elem.h"
+#include "libmesh/fe_base.h"
+#include "libmesh/fem_context.h"
+#include "libmesh/point.h"
+#include "libmesh/quadrature.h"
+
+// Local includes
+#include "poisson.h"
+
+// Bring in everything from the libMesh namespace
+using namespace libMesh;
+
+// Last modified: Feb 13, 2009
+
+// Define the postprocess function to compute QoI 0, the integral of the the solution
+// over a subdomain
+
+void PoissonSystem::element_postprocess (DiffContext &context)
+
+{
+  FEMContext &c = cast_ref<FEMContext&>(context);
```

Comment:
I dont see a operator_spacing.pl in the contrib/bin folder:

vikram@scott:(adjoints_ex_6) /workspace/vikram/libmesh/contrib/bin
$ ls
create_example_docs.sh     detab.sh    generate_reference_elem.sh  reindent.sh              strip_dup_libs.pl           test_installed_headers.sh  update_header_includes.sh
create_libmesh_release.in  ex2html.sh  libmesh-config.in           strip_dup_incl_paths.pl  test_installed_examples.sh  test_src.sh                upload_documentation